config/core/configmaps/deployment.yaml

```diff
@@ -77,3 +77,8 @@ data:
     # for the queue proxy sidecar container.
     # If omitted, no value is specified and the system default is used.
     queueSidecarEphemeralStorageLimit: "1024Mi"
+
+    # queueOnNoMoreTrafficEndpoint is the service endpoint that queue-proxy calls when its traffic
+    # drops to zero or scales up from zero.
+    # If omitted, queue proxy takes no action (this is the default behavior).
+    queueOnNoMoreTrafficEndpoint: ""
```

Comment:
weird my comment was lost


I think we should make this property name more generic and make sure in the future if we need to send additional events we don't need to introduce new options

Reply:
Maybe `concurrencyStateEndpoint`?

Reply:
let's also add a nice big warning that this is experimental/alpha!

Reply:
> Maybe `concurrencyStateEndpoint`?